tendrl/gluster_integration/sds_sync/vol_utilization.py

```diff
@@ -21,15 +21,18 @@ def computeVolumeStats(data):
 
 def showVolumeUtilization(vnames):
     volumes = vnames.split(',')
-    ret_dict = {}
+    ret_dict = {'error' : [], 'success' : []}
```

Comment:
Why error and success have to be list. I see at line#30 you create a dict with `volume` as key and error message as value. So ideally with this logic `ret_dict["error"]` will be a list of dictionaries with one keyvalue pair. Thats not clean I feel.

Reply:
dict created at line #30 is a single volume related error dict that has volume as key and error as message. here the need is to have multiple such entries in case of errors in multiple volume. for this list seemed favorable. in case we go with dict as value to key - 'error', we end up creating multiple levels of nested dict. With dict as list items we have flexibility to store each volume related information in a simpler and easily accessible way. 
Also, if I try to assign the dict at line #30 to 'error', new dict item will replace the old one and we will only have a single entry in the end i.e. one value for one key.
Am I thinking right or am I complicating it? 
@anmolsachan what do you say?